colors/nord.vim

```diff
@@ -72,6 +72,14 @@ let s:nord3_gui_brightened = [
   \ "#7b88a1",
 \ ]
 
+function! NordPalette() abort
+  let ret = {}
+  for color in range(16)
+    execute 'let ret["nord'.color.'"] = s:nord'.color.'_gui'
+  endfor
+  return ret
```

Comment:
We should include the [brightness-increased `nord3` variable][1] to ensure users can make use of it for elements like comments:

```suggestion
  let ret["nord3_bright"] = s:nord3_gui_bright
  return ret
```

[1]: https://github.com/arcticicestudio/nord-vim/blob/ea7ff9c34/colors/nord.vim#L23

Reply:
Makes sense. Wasn't sure if I should include it since it isn't an official Nord color.

Reply:
You're right, but since this function is scoped to the Vim port, and the “special“ color is supported here, it should also be available for usage in the context of Vim scripts.